app/policies/organism_policy.rb

```diff
@@ -4,4 +4,8 @@ def resolve
       scope.all
     end
   end
+    def show?
+    true
```

Comment:
Is this correct? Or should it depend on the user?

Reply:
Yes, because every logged-in user should be able to see all organisms.